projects/demo/src/app/modules/example-primary-tabs-icons/angular-logo.component.ts

```diff
@@ -1,11 +1,11 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
-import {TuiSvgModule} from '@taiga-ui/core';
-import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
+import {TuiSvgComponent} from '@taiga-ui/legacy';
+import {PolymorpheusComponent} from '@taiga-ui/polymorpheus';
 
 @Component({
     standalone: true,
     selector: 'angular-logo',
-    imports: [TuiSvgModule],
+    imports: [TuiSvgComponent],
     template: `
         <tui-svg src="assets/icons/angular.svg"></tui-svg>
```

Comment:
Changer it to `img`, get rid of `tui-svg` everywhere to drop `TUI_SANITIZER` and dompurify

Reply:
Done in https://github.com/taiga-family/maskito/pull/1390/commits/cb38567fbed4c02079bff4493fc3f70ff69b983a